### ISSUE TITLE
Applicability to Legislative and Judicial Branches

### ISSUE THREAD

**Issue by bobmorning** [WhiteHouse/source-code-policy#276]
Subject line is fairly self-explanatory.  Is the federal open source policy applicable to the other branches of government or is this only an executive branch initiative?

**Comment by alex** [comment]
No, this policy is applies to the executive branch exclusively, the executive branch cannot create policy for the other co-equal branches of government.
